src/core_plugins/kibana/public/visualize/editor/editor.js

```diff
@@ -237,6 +237,7 @@ function VisEditor($scope, $route, timefilter, AppState, $window, kbnUrl, courie
     $state.vis.title = savedVis.title;
     $state.vis.type = savedVis.type || $state.vis.type;
     savedVis.visState = $state.vis;
+    savedVis.visType = $state.vis.type;
```

Comment:
This is going to fall apart with point-series charts, because the `.type` changes as people need different configuration.

I think we (@kibana-visualizations) need to provide some sugar around this, something like `vis.getType()`, so we can save this with something meaningful.

Reply:
I can also just interpret the different types I see as `point`. My purpose is to be able to aggregate for the number of different types, and I can add a step in my code that adds line charts, area charts, bar charts, etc as a type called `point-series`

Reply:
Actually, I think we would fulfill both of our requirements if we save the type as `point-series` for all these vis types at index time.